lang/cs/Org.Apache.REEF.Client.Tests/MultiRuntimeConfigureationBuilderTEsts.cs

```diff
@@ -0,0 +1,227 @@
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+
+using Org.Apache.REEF.Client.Multi;
+using Org.Apache.REEF.Common.Runtime;
+
+using Xunit;
+
+namespace Org.Apache.REEF.Client.Tests
+{
+    public class MultiRuntimeConfigureationBuilderTests
+    {
+        [Fact]
+        public void ThrowsOnNotSupportedRuntimes()
+        {
+            MultiRuntimeConfigurationBuilder mrcb = new MultiRuntimeConfigurationBuilder();
+            Exception ex = null;
+            try
+            {
+                mrcb.AddRuntime(RuntimeName.Mesos);
+            }
+            catch (Exception e)
+            {
+                ex = e;
+            }
+
+            Assert.NotNull(ex);
+        }
+
+        [Fact]
+        public void ThrowsWhenMoreThenOneRuntimeAndNoDefault()
+        {
+            MultiRuntimeConfigurationBuilder mrcb = new MultiRuntimeConfigurationBuilder();
+            Exception ex = null;
+            try
+            {
+                mrcb.AddRuntime(RuntimeName.Local);
+                mrcb.AddRuntime(RuntimeName.Yarn);
+                mrcb.SetSubmissionRuntime(SubmissionRuntimeName.YarnRest);
+                mrcb.Build();
+            }
+            catch (Exception e)
+            {
+                ex = e;
+            }
+
+            Assert.NotNull(ex);
+        }
+
+        [Fact]
+        public void NotThrowsWhenMoreThenOneRuntimeAndNoDefault()
+        {
+            MultiRuntimeConfigurationBuilder mrcb = new MultiRuntimeConfigurationBuilder();
+            mrcb.AddRuntime(RuntimeName.Local);
+            mrcb.SetSubmissionRuntime(SubmissionRuntimeName.YarnRest);
+            mrcb.SetMaxEvaluatorsNumberForLocalRuntime(1);
+            mrcb.Build();
+        }
+
+        [Fact]
+        public void ThrowsWhenNoSubmissionRuntimeIsSet()
+        {
+            MultiRuntimeConfigurationBuilder mrcb = new MultiRuntimeConfigurationBuilder();
+            Exception ex = null;
+            try
+            {
+                mrcb.AddRuntime(RuntimeName.Local);
+                mrcb.Build();
+            }
+            catch (Exception e)
+            {
+                ex = e;
+            }
+
+            Assert.NotNull(ex);
+        }
+
+        [Fact]
+        public void ThrowsWhenMaxEvaluatorsNotSetForLocalRuntime()
+        {
+            MultiRuntimeConfigurationBuilder mrcb = new MultiRuntimeConfigurationBuilder();
+            Exception ex = null;
+            try
+            {
+                mrcb.AddRuntime(RuntimeName.Local);
+                mrcb.SetSubmissionRuntime(SubmissionRuntimeName.YarnRest);
+                mrcb.Build();
+            }
+            catch (Exception e)
+            {
+                ex = e;
+            }
+
+            Assert.NotNull(ex);
```

Comment:
check for the exception type, expect the exception via xUnit and add an error message.